### PR TITLE
Okteto Enterprise 0.13.0 release notes

### PR DESCRIPTION
**Pull request by maroshii**
Okteto Enterprise 0.13 Release Notes. Reviewers taken from the changelog between 0.12 and 0.13. Please review in case there's something missing about your changes that should be added (or removed)

### CODE REVIEW

**Review by pchico83 — APPROVED** [review]

File: src/content/enterprise/install/releases.mdx [hunk 1]
```diff
@@ -5,6 +5,28 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 0.13.0
+TBD
+
+### Features
+- Add Access Control to [Volume Snapshots](/docs/enterprise/administration/volume-snapshots/)
+- Add link to Community [Community](https://community.okteto.com) in help menu.
+- Inject OKTETO_NAMESPACE and OKTETO_DOMAIN environment variables in every pod
```
Comment (RinkiyaKeDad) [inline src/content/enterprise/install/releases.mdx:14]:
```suggestion
- Inject `OKTETO_NAMESPACE` and `OKTETO_DOMAIN` environment variables in every pod
```

File: src/content/enterprise/install/releases.mdx [hunk 2]
```diff
@@ -5,6 +5,28 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 0.13.0
+TBD
+
+### Features
+- Add Access Control to [Volume Snapshots](/docs/enterprise/administration/volume-snapshots/)
+- Add link to Community [Community](https://community.okteto.com) in help menu.
```
Comment (RinkiyaKeDad) [inline src/content/enterprise/install/releases.mdx:13]:
```suggestion
- Add link to the [Community website](https://community.okteto.com) in the help menu.
```

File: src/content/enterprise/install/releases.mdx [hunk 3]
```diff
@@ -5,6 +5,28 @@ sidebar_label: Release Notes
...
+- Inject OKTETO_NAMESPACE and OKTETO_DOMAIN environment variables in every pod
+- Remove Namespace limit on licenses
+
+### Improvements
+- Upgrade Okteto CLI to [2.3.3](https://github.com/okteto/okteto/releases/tag/2.3.3)
+- Upgrade registry to [2.8.1](https://github.com/distribution/distribution/releases/tag/v2.8.1)
```
Comment (RinkiyaKeDad) [inline src/content/enterprise/install/releases.mdx:19]: This is Okteto registry right?

```suggestion
- Upgrade Okteto registry to [2.8.1](https://github.com/distribution/distribution/releases/tag/v2.8.1)
```

**Review by RinkiyaKeDad — APPROVED** [review]

File: src/content/enterprise/install/releases.mdx [hunk 4]
```diff
@@ -5,6 +5,28 @@ sidebar_label: Release Notes
...
+- Better deduplication in secrets query. Prioritize user secrets over cluster secrets.
+- Handle PVC update in the mutating webhook. Fixes Okteto CLI issue [#2599](https://github.com/okteto/okteto/issues/2599)
```
Comment (ifbyol) [inline src/content/enterprise/install/releases.mdx:21]: Shouldn't this be in bugfixes?

File: src/content/enterprise/install/releases.mdx [hunk 5]
```diff
@@ -5,6 +5,28 @@ sidebar_label: Release Notes
...
+## 0.13.0
+TBD
+
+### Features
+- Add Access Control to [Volume Snapshots](/docs/enterprise/administration/volume-snapshots/)
+- Add link to Community [Community](https://community.okteto.com) in help menu.
+- Inject OKTETO_NAMESPACE and OKTETO_DOMAIN environment variables in every pod
+- Remove Namespace limit on licenses
+
+### Improvements
```
Comment (ifbyol) [inline src/content/enterprise/install/releases.mdx:17]: Should we include the change to disable autoscrolling in the logs?

Comment (maroshii) [reply src/content/enterprise/install/releases.mdx:17]: I never understood that issue/fix. I thought it was transparent to the user, or is it a bug fix? cc @viddo

File: src/content/enterprise/install/releases.mdx (hunk 4 above)
Comment (maroshii) [reply src/content/enterprise/install/releases.mdx:21]: The bug is in the cli not the backend. I don't think this is fixing an issue in OE itself but rather an improvement that fixes a cli issue

**Pull request merged**